pkg/controller/migmigration/restore.go

```diff
@@ -644,6 +649,77 @@ func (t *Task) deleteMigrated() error {
 	return nil
 }
 
+func (t *Task) deleteDeploymentConfigLeftoverPods() error {
+	// DeploymentConfigs are an exception to the general policy of rollback deleting everything with
+	// the label "migrated-by-migplan: migplan-uid" because DCs spawn additional Pods without
+	// ownerRefs that will mount PVCs. When we delete the DCs, it doesn't cascade to the
+	// deployer and hooks Pods that the DC created, and those Pods sometimes stop PVCs from termianting.
```

Comment:
If explicit hook pod deletion isn't needed here, it might be better to remove the comment about it.